lib/cfg.py

```diff
@@ -348,23 +351,35 @@ def cfg_altiumWorkspace(workspace_file_path, csv_file, bom_search_dir,
         bom_name = "%s%s%s" % (bom_prefix, basename, bom_postfix)
         path_file = os.path.join(file_to_merge_path, basename)
         merge_file_item = os.path.join(path_file, bom_name) + '.csv'
```

Comment:
Il programma deve funzionare sia con i csv che con i file excel. Per sapere quando siamo in un caso o in un altro utilizzare la variabile `csv_file`

Reply:
Aggiunger un test specifico per i due casi, creando dentro la cartella `test` un workspace completo (vedi altri file nei test) e aggiungere due progetti uno con csv e uno con excel.